tests/testthat/test-derive_param_computed.R

```diff
@@ -343,15 +343,7 @@ test_that("derive_param_computed Test 8: no new observations if a constant param
         AVALU = "kg/m2"
       )
     ),
-    regexp = paste(
-      paste(
-        "The input dataset does not contain any observations fullfiling the filter",
-        "condition (NULL) for the parameter codes (PARAMCD) `HEIGHT`"
```

Comment:
![image](https://github.com/pharmaverse/admiral/assets/43018762/12f27321-abb6-48cf-aa37-c7bb4b761aa3)

Background:

- One of R version 4.4's biggest changes is that `is.atomic(NULL)` now returns `FALSE`, it currently returns `TRUE`.
- In our codebase, for `get_hori_data()` which is used for `derive_param_computed()`, in the section that creates warning messaging, we use a function from `rlang::expr_label()` to assist us with the aesthetically pleasing looking warning messages. 
- `expr_label()` is currently under the lifecycle's `"questioning"` label which means `rlang` themselves had thoughts of:  "a function is no longer certain that a function is the optimal approach, but doesn’t yet know how to do it better"
- We would just need to add ticks around NULL to fix/make sure the test passes for 4.4 **but** it would break the tests for 4.3
- For the purpose of **_our_** code/test in this case its really no big deal, I suggest we just lighten up the restriction not to string match the entire warning message, and just the part that says: "The input dataset does not contain any observations fullfiling the filter" so that it would continue to work for all versions

Reply:
@pharmaverse/admiral